src/client/Client.cc

```diff
@@ -6931,13 +6943,9 @@ int Client::_readdir_get_frag(dir_result_t *dirp)
   req->set_filepath(path); 
   req->set_inode(diri.get());
   req->head.args.readdir.frag = fg;
-  if (dirp->last_name.length()) {
+  if (dirp->last_name.length())
```

Comment:
I know it's more verbose, but in general we really prefer to use braces even for one-line if functions; it makes updating them much safer and noticing control flows a lot more obvious.